apps/blog/src/pages/posts/2022-08-29-relaunching-on-astro.md

```diff
@@ -190,7 +192,7 @@ import { typography } from '../utils/typography'
 <!-- ... -->
 
 <!-- Typography -->
-<style is:inline set:html={typography.toString()}>
+<style is:inline set:html={typography.toString()}></style>
```

Comment:
https://github.com/ota-meshi/eslint-plugin-astro/issues/99

This was causing a big issue 😬